pkg/openslo/v1/duration_shorthand.go

```diff
@@ -0,0 +1,113 @@
+package v1
```

Comment:
I wonder if we should move this duration shorthand to separate package in openslosdk. This is a thing that may be helpful for people doing things with OpenSLO. Another thing is that having separate package for this functionality would let us use it with naming convention commonly proposed by go community like this:
``` go
durationshorthand.New(...)

durationshorthand.Parse(...)

// etc
```
I would also make a lot of names much shorter in here

Reply:
it would need to be `durationshorthandv1` though, we don't want it to be a generic package as it is tied to specific OpenSLO version, I think it's worth considering overall :)

Reply:
Yeah, this was my concern too, how to deal with versioning. This makes me think about our project structure, but lets stick to what we have for now. Developing project further will show us what is the best and we can leverage that we are still in WIP phase and we can introduce breaking changes.